airflow.cfg

```diff
@@ -1019,7 +1019,7 @@ zombie_detection_interval = 10.0
 # will not do scheduler catchup if this is ``False``,
 # however it can be set on a per DAG basis in the
 # DAG definition (catchup)
-catchup_by_default = True
+catchup_by_default = False
```

Comment:
I was happy to find this setting! I don't think we've ever wanted a catchup to occur, so seems like a good idea to nix that default.